src/index.html

```diff
@@ -11,9 +11,9 @@
   <body>
     <main class="main">
       <section class="watch">
-        <div class="watch__hand watch__hand--second"></div>
-        <div class="watch__hand watch__hand--minute"></div>
-        <div class="watch__hand watch__hand--center"></div>
+        <div class="watch__arrow watch__arrow--second"></div>
+        <div class="watch__arrow watch__arrow--minute"></div>
+        <div class="watch__arrow watch__arrow--center"></div>
```

Comment:
I meant only center isn't arrow/hand, it's a dot. hand was ok for second/minute

Reply:
aaa, alright, got it